src/test/unit/views/media-account-confirmation.test.ts

```diff
@@ -0,0 +1,157 @@
+import {app} from '../../../main/app';
+import request from 'supertest';
+import {expect} from 'chai';
+import sinon from 'sinon';
+import {MediaAccountApplicationService} from '../../../main/service/mediaAccountApplicationService';
+import {request as expressRequest} from 'express';
+
+let htmlRes: Document;
+
+describe('Media Account Confirmation Page', () => {
+
+  const applicationId = '1234';
+
+  const PAGE_URL = '/media-account-approval?applicantId=' + applicationId;
+  const panelClass = 'govuk-panel__body';
+  const tableHeader = 'govuk-table__header';
+  const tableCell = 'govuk-table__cell';
+  const bottomHeaderClass = 'govuk-heading-m';
+  const bottomContentClass = 'govuk-body';
+
+  const expectedPanel = 'Application has been approved';
+  const nameHeader = 'Name';
+  const nameValue = 'Test Name';
+  const emailHeader = 'Email';
+  const emailValue = 'a@b.com';
+  const employerHeader = 'Employer';
+  const employerValue = 'employer';
+  const appliedHeader = 'Date applied';
+  const appliedValue = '09 May 2022';
+  const proofOfIdHeader = 'Proof of ID';
+  const proofOfIdValue = 'ImageName.jpg (opens in new tab)';
+  const proofOfIdView = 'View';
+  const proofOfIdViewLink = '/media-account-review/image?imageId=12345&applicantId=1234';
+  const bottomHeader = 'What happens next';
+  const bottomContent = 'This account will be created and the applicant will be notified to set up their account. If an account already exists the applicant will be asked to sign in, or choose forgot password.';
+
+  const dummyApplication = {
+    'id': '1234',
+    'fullName': 'Test Name',
+    'email': 'a@b.com',
+    'employer': 'employer',
+    'image': '12345',
+    'imageName': 'ImageName.jpg',
+    'requestDate': '09 May 2022',
+    'status': 'PENDING',
+    'statusDate': '2022-05-09T00:00:01',
+  };
```

Comment:
This dummyApplication object is repeated in a few of the test files, maybe we could add a helper file for some of the consts used in multiple files to DRY out the code a bit?

Reply:
Good point :) I've moved this into a consts file